app/main/views/briefs.py

```diff
@@ -151,12 +149,43 @@ def update_brief_add_user(brief_id, user):
     return jsonify(briefs=brief.serialize(with_users=True)), 200
 
 
+@main.route('/briefs/<int:brief_id>/suppliers/<int:supplier_code>', methods=['PUT'])
+def update_brief_add_rfx_seller(brief_id, supplier_code):
+    updater_json = validate_and_return_updater_request()
+    brief = briefs.get_matching_brief(brief_id)
+
+    # Find the supplier that matches the inputted code, otherwise 404.
+    supplier = suppliers.get_supplier_by_code(supplier_code)
+
+    if not supplier:
+        raise ValidationError("No supplier found: " + supplier_code)
+    # Assign the current seller data to separate variable.
+    # Add the found supplier to said variable.
+    # Amend the brief's data to reflect the added supplier.
+    sellers = brief.data['sellers']
+    sellers[str(supplier_code)] = {'name': supplier.name}
+    brief.data['sellers'] = sellers
```

Comment:
Is it better if we simplify the above three lines to one line?
```python
brief.data['sellers'][str(supplier_code)] = {'name': supplier.name}
```

Reply:
I had attempted to do something very similar to this (and again using the code supplied). It gives the banner message that the brief had been updated, but it did not update the brief. I believe it had something to do with passing by reference, rather than passing the actual object.